firefox-ios/firefox-ios-tests/Tests/ClientTests/Coordinators/HomepageCoordinatorTests.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import ComponentLibrary
+@testable import Client
+
+class HomepageCoordinatorTests: XCTestCase {
+    var coordinator: HomepageCoordinator!
+    let profile = MockProfile()
+    let wallpaperManger = WallpaperManagerMock()
+    let router = MockRouter(navigationController: MockNavigationController())
```

Comment:
Just some general comments about testing; it's preferred to create and deinit variables and test dependencies for each test cases (respectively in `setUp` and `tearDown`). 

> Each test method runs in a separate instance of it's `XCTestCase` subclass. These instances live inside a collection of all tests, which the test runner is iterating over. So all tests cases exists before test execution and are never deallocated until the test runner terminates. This means it's important to use `setUp` and `tearDown` methods to respectively create and nullify any stored properties in our XCTestCase subclasses. 

[[quote from here]](https://github.com/mozilla-mobile/firefox-ios/wiki/Why-and-how-to-write-unit-tests#lifecycle)

Reply:
Since we haven't reached a consensus via slack @lmarceau, I just went ahead and nil-ed everything out.